CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
-## Unreleased
+## 0.16.0 - 2021-02-23
 ### Added
 - Now depends on the "Azure Resources" extension, which provides a "Resource Groups" and "Help and Feedback" view
+- "Attached Emulator..." option added to Attach Database Accounts contest menu
```

Comment:
context* menu

But I'm not sure if this is something that we need to include in the "Added" section.

Reply:
Rather than putting it in the "Changed" section, I meant I'm not sure this constitutes as a feature/enhancement that belongs in the `CHANGELOG`.  It's a pretty low impact change and we usually reserve the CHANGELOG for bigger features/enhancements we want people to discover.